.travis.yml

```diff
@@ -1,24 +0,0 @@
-language: cpp
```

Comment:
I think it's fine if you delete it, we can re-add it if we merge the OS X testing stuff.  I just found out that caching isn't available on travis for public OS X builds, so each build takes about 45 minutes...  Until they fix that, it's pretty worthless.